manageiq-operator/api/v1alpha1/helpers/miq-components/httpd_conf.go

```diff
@@ -92,6 +92,23 @@ func httpdAuthenticationConf(spec *miqv1alpha1.ManageIQSpec) string {
 	}
 }
 
+func httpdHealthConf() string {
```

Comment:
Would it be possible to also include this in the image as a predefined conf file? Then we wouldn't need to generate it on the fly and it would live with the Dockerfile changes in that repo.

Reply:
Sounds like we should really mount a subdir (would also allow us to mount _other_ directories if needed), but it's not a huge priority.